src/ol/featureloader.js

```diff
@@ -129,5 +171,5 @@ ol.featureloader.xhr = function(url, format) {
        */
       function(features) {
         this.addFeatures(features);
-      });
+      }, ol.nullFunction);
```

Comment:
should we add a FIXME here, since failure is not really handled in this case, but it also wasn't before this PR? But we did get rid of the FIXME in this PR.